frostsnap_core/tests/endtoend.rs

```diff
@@ -348,11 +344,19 @@ fn test_display_backup() {
             (decoded.share_index, decoded.secret_share)
         })
         .collect::<Vec<_>>();
-    let interpolated_joint_secret =
-        schnorr_fun::fun::poly::scalar::interpolate_and_eval_poly_at_0(decoded_backups);
 
+    let threshold_interpolated_joint_secret =
+        schnorr_fun::fun::poly::scalar::interpolate_and_eval_poly_at_0(
+            decoded_backups[..(threshold as usize)].to_vec(),
```

Comment:
:bulb: Can you make the test just randomly choose two signers and makes them display the backup and then interpolate. This will be better than fixed indices.